### PR TITLE
Properly Placed Pages: Tags

### PR DESCRIPTION
**Pull request by danielebogo**
This PR adds the tags *Private*, *Pending Review*, *Local Changes* to the Page cells.

ppp-tag

The `Page` model has been modified adding 3 new computed properties, used to get the current state (if it's private or in pending review) or if it can display some tag.
The Page cell will change height (60pt) if the Page has some tag to display.

**To Test:**
- Change the Page state to Private or Pending Review
- Do some local change with no internet connection

### CODE REVIEW

**Comment by loremattei** [comment]
Hey! I moved the milestone to 11.1 since 11.0 has been frozen. If this PR has to land 11.0, please feel free to target the release branch and ping me to build a new beta version.

File: WordPress/Classes/ViewRelated/Pages/PageListViewController.swift [hunk 1]
```diff
@@ -255,6 +255,11 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
         return 0.0
     }
 
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let page = pageAtIndexPath(indexPath)
+        return page.canDisplayTags ? 60.0 : 44.0
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Pages/PageListViewController.swift]: Could these values be constants like `pageSectionHeaderHeight` and `pageCellEstimatedRowHeight`?  Actually, we probably want to use `pageCellEstimatedRowHeight` which has a height of `47.0` vs the `44.0` here.

File: WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m [hunk 2]
```diff
@@ -44,7 +51,13 @@ - (void)applyStyles
     
     self.titleLabel.textColor = [WPStyleGuide darkGrey];
     self.menuButton.tintColor = [WPStyleGuide greyLighten10];
-    
+
+    self.privateBadgeLabel.text = NSLocalizedString(@"Private", @"Title of the Private Badge");
+    self.localChangesLabel.text = NSLocalizedString(@"Local changes", @"Title of the Local Changes Badge");
+
+    self.privateBadge.layer.cornerRadius = 2.0;
+    self.localChangesBadge.layer.cornerRadius = 2.0;
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m]: Should we have a constant for the corner radius?

**Comment by SylvesterWilmott** [comment]
>I notice that these labels have a background where the ones in the post list do not (tho those have an icon)

There were some considerations for this when we did the labels on Android:

- Post list uses icon and text which adds a lot of space if we are stacking two labels in the case of: [icon] Private [separating space] [icon] Local changes, this extra space + the indenting that we use on the pages list can easily knock a label off the edge of the cell container.
- Calypso uses a prefix eg: `Private: Page name`. Again, same thing with the lack of horizontal space on phones, it can easily truncate the page title more than it needs to be.
- Tag design with background colour: They can sit very close to each other and you can still distinguish between them.

>I also wonder if it would be good to align the label text with the page title?

Good spot, can we try this @danielebogo ?

**Comment by danielebogo** [comment]
simulator screen shot - iphone x - 2018-10-11 at 10 23 13
simulator screen shot - iphone x - 2018-10-11 at 10 23 16

Here @SylvesterWilmott

**Comment by danielebogo** [comment]
@aerych so I fixed everything you comment, including the *Draft* bug you found 👍

**Comment by SylvesterWilmott** [comment]
Looking good with the updated alignment!

**Pull request merged**